data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,29 @@ mission "Station Exploration Abandon Ship"
 			label fine
 			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
 
+
+
+mission "Keystone Registration"
+	invisible
+	source
+		government "Hai"
+		not attributes "uninhabited"
+	to offer
+		or
+			has "outfit: Quantum Keystone"
+			has "outfit: Quantum Key Stone"
+			has "outfit: Shield Beetle Pendant"
+			has "outfit: Tree Skeleton Key Stone"
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`Upon landing, you receive a message from the local port authority. "Hello, Captain <last>. We have been informed that you have recently installed a keystone, or its equivalent, onto your ship. We have registered you with the job board, which will allow you to take passengers that specifically request a keystone-equipped ship." Once the message ends, you can hear a quiet tapping on the door of your ship's hatch. You turn to see a short Hai in a spaceport uniform peeking through your window. He waves and smiles.`
```

Comment:
the jobs are intended to only accept the actual hai keystone, iirc?

Reply:
Changed to have the message mention this. Perhaps we should add jobs that pay less but are less picky about what kind of keystone the player has?